src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,66 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = Array.from({ length: totalPages }, (_, i) => i + 1);
+
+  const handlePageChange = (newPage: number) => {
+    if (newPage !== currentPage) {
+      onPageChange(newPage);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
```

Comment:
or even better to use class names
```suggestion
      <li className={`page-item ${currentPage === 1 && 'disabled' }`}>
```